WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -117,59 +117,54 @@ private extension StoreStatsAndTopPerformersViewController {
             }
         }
 
-        // Loads site for the latest site timezone.
-        let loadSiteAction = AccountAction.loadSite(siteID: siteID) { [weak self] site in
-            guard let site = site, let self = self else {
-                return
-            }
-            let timezone = site.timezone
-            let siteTimezone = TimeZone(identifier: timezone) ?? TimeZone(secondsFromGMT: 0) ?? TimeZone.current
-            self.periodVCs.forEach { (vc) in
-                vc.siteTimezone = siteTimezone
-
-                let currentDate = Date()
-                vc.currentDate = currentDate
-                let latestDateToInclude = vc.timeRange.latestDate(currentDate: currentDate, siteTimezone: siteTimezone)
-
-                group.enter()
-                self.syncStats(for: siteID,
-                               siteTimezone: siteTimezone,
-                               timeRange: vc.timeRange,
-                               latestDateToInclude: latestDateToInclude) { [weak self] error in
-                    if let error = error {
-                        DDLogError("⛔️ Error synchronizing order stats: \(error)")
-                        syncError = error
-                    } else {
-                        self?.trackStatsLoaded(for: vc.granularity)
-                    }
-                    group.leave()
+        // When the site time zone can be correctly fetched, and also supports the Stats v4 API, consider using the site time zone
+        // for stats UI (#1375).
+        let timezoneForStatsUI = TimeZone.current
```

Comment:
I assume I just can not see it, but it seems like the comment does not match the code? I mean, it seems like we always use the device timezone, what am I missing?

Reply:
ah `timezoneForStatsUI` is a bad name now I look at it! this time zone is what the UI is going to display the dates in. Before this PR, we display the stats in the site time zone by GMT dates from the API + site time zone in this line of code for GMT offset.

Now that we're not using GMT dates, there's no need to offset the dates. Thus, passing the current time zone here.

Reply:
I renamed it to `timezoneForStatsDates`, hopefully it matches what it is used for more!